packages/app/src/app/pages/Sandbox/Editor/Header/AppMenu/formatMenuData.ts

```diff
@@ -0,0 +1,78 @@
+import { MenuAppItems } from 'app/overmind/effects/vscode/composeMenuAppTree';
+
+// TODO: find out a proper place to TS helpers
```

Comment:
Is there any proper file to place global helpers?

Reply:
I don't think we have a place right now, @christianalfoni do you know?